<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScriptEditWidget</class>
 <widget class="QWidget" name="ScriptEditWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>272</width>
    <height>457</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1</width>
    <height>1</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0" colspan="6">
    <widget class="QLineEdit" name="m_filenameEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="6">
    <widget class="QSplitter" name="m_splitter">
     <property name="minimumSize">
      <size>
       <width>1</width>
       <height>1</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="Lua::LuaEditor" name="m_scriptEdit">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>2</verstretch>
       </sizepolicy>
      </property>
      <property name="tabStopDistance">
       <number>20</number>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="m_printOutputEdit">
      <property name="undoRedoEnabled">
       <bool>false</bool>
      </property>
      <property name="lineWrapMode">
       <enum>QPlainTextEdit::NoWrap</enum>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="plainText">
       <string notr="true"/>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
      </property>
      <property name="centerOnScroll">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </item>
   <item row="0" column="0" colspan="6">
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QToolButton" name="m_openButton">
        <property name="toolTip">
         <string>Open</string>
        </property>
        <property name="text">
         <string>open</string>
        </property>
        <property name="icon">
         <iconset resource="../qosmic.qrc">
          <normaloff>:/icons/silk/folder.xpm</normaloff>:/icons/silk/folder.xpm</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>18</width>
          <height>18</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="m_saveButton">
        <property name="toolTip">
         <string>Save</string>
        </property>
        <property name="text">
         <string>save</string>
        </property>
        <property name="icon">
         <iconset resource="../qosmic.qrc">
          <normaloff>:/icons/silk/disk.xpm</normaloff>:/icons/silk/disk.xpm</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>18</width>
          <height>18</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="m_saveAsButton">
        <property name="toolTip">
         <string>Save As</string>
        </property>
        <property name="text">
         <string>save as</string>
        </property>
        <property name="icon">
         <iconset resource="../qosmic.qrc">
          <normaloff>:/icons/silk/page_save.xpm</normaloff>:/icons/silk/page_save.xpm</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="m_runButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Run</string>
        </property>
        <property name="text">
         <string>run</string>
        </property>
        <property name="icon">
         <iconset resource="../qosmic.qrc">
          <normaloff>:/icons/silk/bullet_go.xpm</normaloff>:/icons/silk/bullet_go.xpm</iconset>
        </property>
        <property name="shortcut">
         <string>Ctrl+Return</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>1</width>
          <height>2</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QToolButton" name="m_configButton">
        <property name="toolTip">
         <string>config</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../qosmic.qrc">
          <normaloff>:/icons/silk/bullet_wrench.xpm</normaloff>:/icons/silk/bullet_wrench.xpm</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Lua::LuaEditor</class>
   <extends>QTextEdit</extends>
   <header>src/lua/luaeditor.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../qosmic.qrc"/>
 </resources>
 <connections/>
</ui>
